samples/servers/mlserver-badname.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Server
```

Comment:
:question: Should there be a notebook accompanying this?

Reply:
Wasn't planning to do a notebook